src/nitro.ts

```diff
@@ -126,7 +126,7 @@ export async function createNitro(
 
   // Virtual module for tasks (TODO: Move to rollup plugin)
   nitro.options.virtual["#internal/nitro/virtual/tasks"] = () => {
-    const _scheduledTasks = Object.entries(nitro.options.scheduledTasks || {})
+    const _scheduledTasks = Object.entries(nitro.options.scheduledTasks)
```

Comment:
there is no harm to have extra runtime safety :) 

```suggestion
    const _scheduledTasks = Object.entries(nitro.options.scheduledTasks || {})
```